Core/Utilities/File.py

```diff
@@ -179,7 +176,7 @@ def getGlobbedTotalSize(files):
   else:
     for path in glob.glob(files):
       if os.path.isdir(path):
-        for content in os.listdir(path):
+        for content in os.listdir(path) and not os.path.islink(path):
```

Comment:
The `and not...` has to be moved to the if statement in the line above.

Reply:
ah yes thanks